doc/Rules.md

```diff
@@ -1598,6 +1600,8 @@ Plain text in a code block
 ```
 ````
 
+You can configure the `allowed_languages` parameter to specify a list of languages code blocks could use. The default value is `null` which means any language specifier is valid.
```

Comment:
I think it will want you to wrap this line at 80 characters?

Reply:
Looks like the tests pass, but it would still be good to do this to be consistent with other text in the file.